package.json

```diff
@@ -37,7 +37,8 @@
         "test:file:fork": "yarn hardhat --config hardhat-fork.config.ts test",
         "test:file": "yarn hardhat test",
         "test:file:long": "LONG_TESTS=true yarn hardhat test",
-        "prepublishOnly": "yarn run compile"
+        "copy-types": "cp -R types/generated dist/types",
+        "prepublishOnly": "yarn run compile && yarn run copy-types"
```

Comment:
Super minor thing but you dont need the `run` with yarn

Reply:
I'll make is consistent with the other scripts and remove the `run`s